tests/pages/leftNavMenu.js

```diff
@@ -101,6 +102,12 @@ module.exports = {
       configurationAdvisors: menuOption(ad, 'Configuration Advisors', '/graph/advisors/configuration'),
       securityAdvisors: menuOption(ad, 'Security Advisors', '/graph/advisors/security'),
     }),
+  backups: new LeftMenu('Backup', '/graph/backup/inventory', {
+    allBackups: menuOption(bm, 'Tab All Backups', '/graph/backup/inventory'),
```

Comment:
Drop the "Tab" pls. I think it has to be exact match to this.
![image](https://user-images.githubusercontent.com/64002801/227513060-14ab1881-c8de-4b1f-a376-8d38a20721d7.png)